### PR TITLE
Update CVN-operators-guide.md

### PR DESCRIPTION
**Pull request by rasos**
Update links, fee, CVN abbreviation

### CODE REVIEW

File: doc/CVN-operators-guide.md [hunk 1]
```diff
@@ -75,14 +80,15 @@ We expect that in the short term, this will not be enough to cover electricity c
 
 When the profit exceeds the investment, the open FairCoop assembly could decide to reduce the fees or to pool a part of the incomes for FairCoin development or infrastructure costs. Also, in the future we may agree on using other methodologies for avoiding spam risks and to cover the CVN expenses.
 
-Fees are mandatory and dynamically adjustable. The effective fee per Kb is decided in the FairCoop assembly. On June 16th 2016, in a corresponding assembly (https://fair.coop/docs/minutes-of-the-10th-faircoop-open-assembly/) it has been decided to start with a fee of 0.1 FAIR per 1000 bytes. This is an open process of discussion where every FairCoin user can be involved.
+Fees are mandatory and dynamically adjustable. The effective fee per Kb is decided in the FairCoop assembly. On June 16th 2016, in a corresponding assembly (https://fair.coop/docs/minutes-of-the-10th-faircoop-open-assembly/) it has been decided to start with a fee of 0.1 FAIR per 1000 bytes. Due to the value increase, the decision has ben revised in the June assembly 2017 (https://board.net/p/r.43161932a89400cc42bdae476265011c) just before the launch of the Proof-of-Cooperation mechanism to 0.01 FAIR.
```
Comment (thokon00) [inline doc/CVN-operators-guide.md]: I think at the end of the paragraph it should state 0.01 FAIR/Kb.

File: doc/CVN-operators-guide.md [hunk 2]
```diff
@@ -20,8 +20,13 @@ A secure and stable operating system is crucial for reliably running a CVN. Ther
 6. The hardware must provide a USB 2.0 port which takes up the Fasito USB token. Please avoid using a USB-hub for the token.
 7. Although not absolutely necessary, a UPS (uninterruptible power supply) would be appreciated. Note that all network related equipment should also be hooked up to the UPS to keep the network connection up in the event of a power outage.
 
+A CVN can run on a RaspBerry 3, however we recommend some more decent hardware.
```
Comment (thokon00) [inline doc/CVN-operators-guide.md]: Official naming is: Raspberry Pi 3

File: doc/CVN-operators-guide.md [hunk 3]
```diff
@@ -75,14 +80,15 @@ We expect that in the short term, this will not be enough to cover electricity c
...
 
 ## 4. Organizational process for running a CVN
 
 ### 4.1  Socio-political criteria for running a CVN
-The person responsible for certification should follow KYCVN criteria (Know your CVN). This means that online or physically, some questions should be asked to get to know the people or groups applying for certification and their real motivations.
+We are performing peer investigations and are following some KYCVN criteria ("Know your CVN") and want to know the motivations. This means that online or physically, some questions are asked to get to know the people or groups applying to run a CVN.
+
+Online information like a website, profiles in social networks like fair.coop, bitcointalk, twitter, facebook, etc. help to find out more about an applicant. ideally, applicants should visit a FairCoop local node nearby or meet a person who is already involved in FairCoin development, commuication or running a CVN. This helps to get to know them better and thus ensure that CVN operators have the right intentions.
```
Comment (thokon00) [inline doc/CVN-operators-guide.md]: ideally -> Ideally

**Review by thokon00 — CHANGES_REQUESTED** [review]

**Review by rasos — COMMENTED** [review]
TNX for reviewing, changes applied.

**Review by thokon00 — APPROVED** [review]

**Comment by thokon00** [comment]
Sorry, I just realised that you created the PR against the unused faircoin2 branch. Please use the master branch instead.

**Pull request closed without merging**